apps/openmw/mwworld/cellstore.cpp

```diff
@@ -685,8 +685,8 @@ namespace MWWorld
     {
         state.mId = mCell->getCellId();
 
-        if (mCell->mData.mFlags & ESM::Cell::Interior && mCell->mData.mFlags & ESM::Cell::HasWater)
-            state.mWaterLevel = mWaterLevel;
+        // if (mCell->mData.mFlags & ESM::Cell::Interior && mCell->mData.mFlags & ESM::Cell::HasWater)
```

Comment:
Why comments instead of deleting?

Reply:
Read first message please, I added info about this. I'm just not sure.

Reply:
I think the correct fix is to remove the unconditional state.mWaterLevel = mWaterLevel; . The loadState function above has the same issue btw.